src/__tests__/Carousel.js

```diff
@@ -293,29 +293,29 @@ describe("Slider", function() {
                 componentInstance.decrement = jest.genMockFunction();
             });
 
-            it('should call only increment on ArrowDown', () => {
-                componentInstance.navigateWithKeyboard({key: 'ArrowDown'});
+            it('should call only increment on 40', () => {
+                componentInstance.navigateWithKeyboard({keyCode: 40});
 
                 expect(componentInstance.increment.mock.calls.length).toBe(1);
                 expect(componentInstance.decrement.mock.calls.length).toBe(0);
             });
 
-            it('should call only decrement on ArrowUp', () => {
-                componentInstance.navigateWithKeyboard({key: 'ArrowUp'});
+            it('should call only decrement on 38', () => {
```

Comment:
Please, keep the tests descriptions as "should call only decrement on ArrowUp (38)"... Much easier to understand.